README.md

```diff
@@ -20,6 +20,18 @@
 
 StableNetGuardOwl provides a robust suite for conducting stability tests on Neural Network Potentials (NNPs) used in molecular simulations. These tests are critical in validating NNPs for research and industrial applications, ensuring accuracy and reliability.
 
+
+## Installation
+
+Since openMM and PhysML use different package managers, obtaining a conda|mamba environment with the correct packages is not trivial.
+The following (note the order of the installation of the packages, this is critical for a working environment) has worked in the past:
+```bash
+mamba create --name owl python=3.11
+mamba activate owl
+pip install "physicsml[openmm]"
+mamba install openmm-ml pytorch-gpu -c conda-forge
+mamba install openmmtools loguru typer
```

Comment:
openff-toolkit is also required

Reply:
and `dvc[s3]` if svc functionality is required

Reply:
Thanks! I have added these packages!